### ISSUE TITLE
error[E0554]: `#![feature]` may not be used on the stable release channel

### ISSUE THREAD

**Issue by meua** [dora-rs/dora#283]
**Describe the bug**
```bash
error[E0554]: `#![feature]` may not be used on the stable release channel
  --> /home/jarvis/.cargo/registry/src/github.com-1ecc6299db9ec823/rustix-0.36.9/src/lib.rs:99:26
   |
99 | #![cfg_attr(rustc_attrs, feature(rustc_attrs))]
   |                          ^^^^^^^^^^^^^^^^^^^^
```

**To Reproduce**
1. Debug start coordinator.

**Environments (please complete the following information):**
 - System info: ubuntu 22.04
 - Dora version: dora-rs/dora main branch

**Comment by sunfishcode** [comment]
This is fixed in rustix 0.36.11 and later, as well as 0.37.5 and later.

**Comment by meua** [comment]
> This is fixed in rustix 0.36.11 and later, as well as 0.37.5 and later.

Thank you very much for the tip, I will try to do what you said.
